spec-tests/src/main/java/org/candlepin/spec/bootstrap/assertions/OnlyWithManifestGen.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2009 - 2023 Red Hat, Inc.
+ *
+ * This software is licensed to you under the GNU General Public License,
+ * version 2 (GPLv2). There is NO WARRANTY for this software, express or
+ * implied, including the implied warranties of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv2
+ * along with this software; if not, see
+ * http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+ *
+ * Red Hat trademarks are not licensed under GPLv2. No permission is
+ * granted to use or replicate Red Hat trademarks that are incorporated
+ * in this software or its documentation.
+ */
+package org.candlepin.spec.bootstrap.assertions;
+
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.condition.EnabledIf;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * This annotation marks a test or test suite to run only in hosted mode.
+ */
```

Comment:
This should be rewritten to look more like the `OnlyWithCapability` annotation (as should `OnlyInHosted` and `OnlyInStandalone`) since `EnabledIf`/`DisabledIf` are implemented so poorly and unreliably.

For now I've just removed it since none of the tests use it and instead rely on manual use of `assumeTrue` for conditional execution.